x/campaign/client/cli/tx_redeem_vouchers.go

```diff
@@ -0,0 +1,69 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/spf13/cast"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/x/campaign/types"
+)
+
+var _ = strconv.Itoa(0)
+
+const (
+	flagAccount = "account"
+)
+
+func CmdRedeemVouchers() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "redeem-vouchers [campaign-id] [vouchers]",
+		Short: "Redeem vouchers",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			argCampaignID, err := cast.ToUint64E(args[0])
```

Comment:
We use the `arg` prefix in Starport templates to avoid as much as possible conflicts with other vars

I think it's visually better to remove those here

Reply:
Sure!! but it's good we put it as an item in the #299 issue because we are using the `arg` prefix in all commands